docs/generate_docs.py

```diff
@@ -393,6 +393,12 @@ def main():
     if "python_backend" in repo_tags:
         clone_from_github("python_backend", repo_tags["python_backend"], github_org)
 
+    # Usage generate_docs.py --repo-tag=tensorrtllm_backend:main
+    if "tensorrtllm_backend" in repo_tags:
```

Comment:
A quick heads up, I think @statiraju is refactoring this section and will get rid of annoying if statements.

Reply:
Thanks. Confirmed that we can merge this one for now, and Suman will add his changes on top of this.